app/controllers/api/episodes_controller.rb

```diff
@@ -3,4 +3,8 @@ class Api::EpisodesController < Api::BaseController
   represent_with Api::EpisodeRepresenter
   filter_resources_by :podcast_id
   find_method :find_by_guid
+
+  def scoped(relation)
+    relation.published.released
```

Comment:
Gotcha.  I was thinking that there would never be anything in the Feeder database that wasn't already "public" in CMS.  But with the ability to unpublish or stage-for-release something, I guess that's not the case.